packages/storybook-react/src/stories/Alert.stories.tsx

```diff
@@ -30,7 +30,7 @@ const meta = {
     docs: {
       page: () => (
         <>
-          <Description markdown={readme} />
+          <Description>{readme}</Description>
           <Primary />
           <ArgsTable story={PRIMARY_STORY} />
           <Stories />
```

Comment:
Alert geeft bij mij een foutmelding bij de rendering van `parameters.docs.page`: `e.background is undefined`